### PR TITLE
Assorted documentation updates

### PR DESCRIPTION
**Pull request by BrentEaston**

### CODE REVIEW

File: vassal-doc/src/main/readme-referencemanual/ReferenceManual/Expression.adoc [hunk 1]
```diff
@@ -244,6 +244,31 @@ Range functions calculate the range in pixels, or in grid units (hexes, squares)
 If a message needs to be shown to _all_ players, the recommended method is to send something to the Chat Log, e.g.,
 with a <<ReportChanges.adoc#top,Report Action>> trait or other <<MessageFormat.adoc#top,Message Format>> field.
 
+|*Sleep* | Sleep(ms)|{Sleep(500)} + |*Sleep* causes the screen to refresh and then pauses all screen updates for the number of milliseconds specifed. The example will 'sleep' for a 1/2 second.
```
Comment (jonathanrwatts) [inline vassal-doc/src/main/readme-referencemanual/ReferenceManual/Expression.adoc:247]: *specified

File: vassal-doc/src/main/readme-referencemanual/ReferenceManual/Expression.adoc [hunk 2]
```diff
@@ -244,6 +244,31 @@ Range functions calculate the range in pixels, or in grid units (hexes, squares)
...
+Audit(text, options) +
+Audit(text, expression) +
+Audit(text, expression, options)
+|
+{Audit("In Trigger 1"} +
+{Audit("Text to Chat only", "CS")} +
+{Audit("Oops", "{Turn == \"X1\"}") +
+{Audit("Oops", "{Turn == \"X1\"}", "C")
+
+|*Audit* writes a message (text) to the errorlog and or the Chat Window. +
```
Comment (jonathanrwatts) [inline vassal-doc/src/main/readme-referencemanual/ReferenceManual/Expression.adoc:260]: *and/or

File: vassal-doc/src/main/readme-referencemanual/ReferenceManual/Expression.adoc (hunk 1 above)
Comment (BrentEaston) [reply vassal-doc/src/main/readme-referencemanual/ReferenceManual/Expression.adoc:247]: Fixed

File: vassal-doc/src/main/readme-referencemanual/ReferenceManual/Expression.adoc (hunk 2 above)
Comment (BrentEaston) [reply vassal-doc/src/main/readme-referencemanual/ReferenceManual/Expression.adoc:260]: Fixed

**Pull request merged**